LttPAdjuster.py

```diff
@@ -897,7 +897,10 @@ def icon_section(self, frame_label, path, no_results_label):
         sprites = []
 
         for file in os.listdir(path):
-            sprites.append((file, Sprite(os.path.join(path, file))))
+            try:
+                sprites.append((file, Sprite(os.path.join(path, file))))
+            except Exception as e:
```

Comment:
Would be good to only catch specific exception here. I think there are some errors (like permission denied) you would want to see immediately?

Reply:
so... i took a look at the `Sprite.__init__`, and i think there are too many code paths that set valid=False instead of throwing for the try-except to be a good solution. I would propose to:
* ```python
  sprite = Sprite(...)
  if sprite.valid:
    sprites.append(sprite)
  ```
* have `from_ap_sprite(self, filedata)` check for errors (like try..except as above) and set valid=False

Reply:
Do you want me to write what i've been suggesting? Don't like being a nay sayer and then not putting in the work.

If yes, should i push to [lttp_adjuster_ignore_sprites](https://github.com/ArchipelagoMW/Archipelago/tree/lttp_adjuster_ignore_sprites), or PR to it?

Reply:
I did just put this together quickly to prevent the UI tearing itself apart as result of user-error of putting broken files in the folder. A better way would alert the user with a popup or such as opposed to just logging the problem away. But, yeah, have at it.